devlab.yml

```diff
@@ -0,0 +1,25 @@
+# DOCKER-VERSION 1.9.1
+# DEVLAB-VERSION 1.5.3
+
+#
+# Stardust DevLab YAML
+#
+# http://fluidbyte.net/docker/devlab/nodejs/2015/10/15/containerize-your-local-dev-in-minutes-with-devlab/
+# https://github.com/TechnologyAdvice/stardust/
+#
+
+from: node:argon
```

Comment:
We have a node:4 container you could use for consistency: technologyadvice/node-build:4 [see hub-template](https://github.com/TechnologyAdvice/hub-template/blob/staging/devlab.yml)